app/gui/src/controller/searcher/component/builder.rs

```diff
@@ -348,6 +362,9 @@ mod tests {
                     #[in_group("First Group")]
                     fn fun2() -> Standard.Base.Any;
 
+                    #[with_doc_section(DocSection::Tag { tag: Tag::Private, body: default() })]
```

Comment:
We have a syntax for tags: `#[with_doc_section(doc_section!(@ Private, ""))]`

Reply:
I looked for one but haven't found it. Thanks! Where is it documented?